From 1efd5c327fdccddc157277d0110bb854ef237d8d Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Sat, 22 Feb 2025 18:26:38 +0100
Subject: [PATCH] Get pkg-config from conda PREFIX

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 0959b178a..cd88a5ead 100644
--- a/setup.py
+++ b/setup.py
@@ -85,7 +85,7 @@
     SUP_EXTERNAL=True
     sys.argv.remove("--external")
 
-pkgconf = "/usr/bin/pkg-config"
+pkgconf = os.path.join(os.environ["BUILD_PREFIX"], "bin", "pkg-config")
 if "--libzstd-bundled" in sys.argv:
     # Do you want use external Zstd library?
     SUP_EXTERNAL=False
